javaslang/generator/Generator.scala

```diff
@@ -147,6 +150,80 @@ def generateMainClasses(): Unit = {
               return $EitherType.left(left);
           }
 
+          /$javadoc
+           * Alias for {@link $FutureType#of($CheckedSupplierType)}
+           *
+           * @param <T>         Type of the computation result.
+           * @param computation A computation.
+           * @return A new Future instance.
+           * @throws NullPointerException if computation is null.
+           */
+          public static <T> $FutureType<T> future($CheckedSupplierType<? extends T> computation) {
+              return $FutureType.of(computation);
+          }
+
+          /$javadoc
+           * Alias for {@link $FutureType#of($ExecutorServiceType, $CheckedSupplierType)}
+           *
+           * @param <T>             Type of the computation result.
+           * @param executorService An executor service.
+           * @param computation     A computation.
+           * @return A new Future instance.
+           * @throws NullPointerException if one of executorService of computation is null.
```

Comment:
`executorService of computation` should probably be `executorService or computation`